deny.toml

```diff
@@ -244,6 +244,8 @@ deny = [
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
     #{ name = "ansi_term", version = "=0.11.0" },
+    { name = "hashbrown", version = "=0.14.5" },
```

Comment:
I think this change should be fine... but I'm highlighting this so it isn't missed.